### PR TITLE
Adding a file-based credentials test. Bumping up the common lib as we…

### PR DESCRIPTION
**Pull request by MushMal**
… had a bug that got fixed to support empty session token.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tst/SignalingApiTest.cpp [hunk 1]
```diff
@@ -48,6 +48,56 @@ TEST_F(SignalingApiTest, signalingSendMessageSync)
...
+    MEMSET(signalingMessage.payload, 'A', 100);
+    signalingMessage.payload[100] = '\0';
+    signalingMessage.payloadLen = 0;
+    signalingMessage.correlationId[0] = '\0';
+
+    EXPECT_EQ(STATUS_SUCCESS, signalingClientConnectSync(mSignalingClientHandle));
+    EXPECT_EQ(STATUS_SUCCESS, signalingClientSendMessageSync(mSignalingClientHandle, &signalingMessage));
+
+    // Some correlation id
+    STRCPY(signalingMessage.correlationId, SIGNAING_TEST_CORRELATION_ID);
```
Comment (disa6302) [inline tst/SignalingApiTest.cpp:85]: Just a typo in the inital defintion I guess. But, if possible, we can correct it in this PR

Comment (MushMal) [reply tst/SignalingApiTest.cpp:85]: I don't see the typo though

Comment (disa6302) [reply tst/SignalingApiTest.cpp:85]: Shouldnt it be SIGNA**L**ING_TEST_CORRELATION_ID? 
I know it is defined as `SIGNAING_TEST_CORRELATION_ID` elsewhere though.

Comment (MushMal) [reply tst/SignalingApiTest.cpp:85]: It does use SIGNAING_TEST_CORRELATION_ID as a test correlation id. Perhaps I am not understanding the question..

**Pull request merged**